src/main/java/teammates/common/util/Url.java

```diff
@@ -2,12 +2,16 @@
 
 public class Url {
 
-    private String urlString;
+    protected String urlString;
 
     public Url(String url) {
-        this.urlString = url;
+        this.urlString = url.replace(getAppUrl(), ""); // force the URL to be relative
     }
```

Comment:
It's not new; it's only making the separation between relative and absolute URLs cleaner, which can be called separately using `toString()` and `toAbsoluteString()`. The class has been behaving that way all along. Maybe not before when there are still instances of `new Url(Config.APP_URL + url)`, but I've removed all instances of that in #4582.
Renaming the class will make so many more changes than necessary.